plugins/package-managers/node/src/funTest/kotlin/NpmFunTest.kt

```diff
@@ -126,9 +126,9 @@ class NpmFunTest : WordSpec({
             val expectedResult = patchExpectedResult(expectedResultFile, definitionFile)
                 .fromYaml<ProjectAnalyzerResult>()
 
-            val actualResult = create("NPM").resolveSingleProject(definitionFile, resolveScopes = true)
+            val result = create("NPM").resolveSingleProject(definitionFile, resolveScopes = true)
```

Comment:
I'm not sure *what* this aligns to (the commit message does not say), but I believe I've shared earlier that personally I'd prefer to align on `actualResult` as it better matches `expectedResult`.